internal/app/components/timelines/timelines.go

```diff
@@ -22,12 +22,23 @@ func newTimelineId() TimelineId {
 	return TimelineId(xid.New().String())
 }
 
-// TODO: use cr parameter when creating Timeline
-func NewTimeline(name TimelineName, cr *courses.CourseRecords) *Timeline {
+func NewTimeline(name TimelineName, courseHistory []*terms.TermRecord) *Timeline {
+	var historyCopy []*terms.TermRecord
+	for _, tr := range courseHistory {
+		uwTermId := int(tr.Term.Season) + (int(tr.Term.Year)-1900)*10
```

Comment:
I think we should add an `Id` field to the `Term` struct, we shouldn't have this kind of logic here.